app/models.py

```diff
@@ -119,16 +125,16 @@ def serialize(self):
             'name': self.name,
             'slug': self.slug,
             'framework': self.framework,
-            'countersignerName': (
-                self.framework_agreement_details or {}
-            ).get("countersignerName"),
-            'frameworkAgreementVersion': (
-                self.framework_agreement_details or {}
-            ).get("frameworkAgreementVersion"),
+            'countersignerName': (self.framework_agreement_details or {}).get("countersignerName"),
+            'frameworkAgreementVersion': (self.framework_agreement_details or {}).get("frameworkAgreementVersion"),
             'variations': (self.framework_agreement_details or {}).get("variations", {}),
             'status': self.status,
             'clarificationQuestionsOpen': self.clarification_questions_open,
             'lots': [lot.serialize() for lot in self.lots],
+            'application_close_date': (
+                self.application_close_date.strftime(DATETIME_FORMAT) if self.application_close_date else ''
```

Comment:
Ah like at 
https://github.com/alphagov/digitalmarketplace-api/blob/master/app/models.py#L497-L497
best to stay consistent, I'll get that changed